src/chigame/games/views.py

```diff
@@ -1,15 +1,24 @@
+from django.shortcuts import render, redirect, reverse
 from django.views.generic import DetailView, ListView
 
-from .models import Game
-
+from .models import *
 
 class GameListView(ListView):
     model = Game
     queryset = Game.objects.all()
     template_name = "games/game_list.html"
 
+def lobby_list(request):
+    lobbies = Lobby.objects.all()
+    context = {"object_list": lobbies}
+    return render(request, "games/lobby_list.html", context)
+
+class ViewLobbyDetails(ListView):
```

Comment:
I think this should inherit from DetailView vs. ListView. That way, if we go to a lobby/pk/ where a lobby with this pk doesn't exist, we get a 404.